src/common/routing/RouteDefs.jsx

```diff
@@ -36,8 +36,12 @@ const RouteDefs = () => {
                 <PlayList/>
               }
             />
-                  <Route exact path="create" element= {<CreatePlay />}/>
-                  <Route exact path="created/:playid" element= {<PlayCreated />}/>
+                  { process.env.NODE_ENV === "development"
+                    && <>
+                      <Route exact path="create" element= {<CreatePlay />}/>
```

Comment:
Hey @6km . I would suggest having the path intact however when someone hits this path on production the page will show a custom message something like `You can only create a play in dev mode. For details visit <md_link>`. This is applicable only for `create` route only.

On the other hand, `created/:playid` will continue to be wrapped withing the "if" condition

Reply:
@koustov done, can you re-review please?